tutorials/math/interpolation.rst

```diff
@@ -118,24 +118,25 @@ Interpolation can be used to smooth movement, rotation, etc. Here is an example
 
     func _physics_process(delta):
         var mouse_pos = get_local_mouse_position()
-
-        $Sprite2D.position = $Sprite2D.position.lerp(mouse_pos, delta * FOLLOW_SPEED)
```

Comment:
I suggest to keep the current version, but to add a comment right after : 
```
// An alternative version with even smoother results with a little more math : 
// $Sprite2D.position = $Sprite2D.position.lerp(mouse_pos, 1 - exp(-FOLLOW_SPEED * delta))
```

Reply:
I don't know. Kind of kills the idea of fixing the incorrect method in the first place.

Reply:
But the original method is not really _wrong_ in itself. It just produces less smooth results, which might not be a problem in the context of a beginners tutorial.